src/Tabs/TabContent.js

```diff
@@ -0,0 +1,36 @@
+import classnames from 'classnames';
```

Comment:
@chrismanciero - the SAP Concur team discussed the pattern to be used for subcomponents, and wanted to use a "private" component structure.

to do this we typically prefix private components with underscores, and import/export them as part of the parent component. Reach to @greg-a-smith or others for more guidance.